services/dy-static-file-server/docker-compose-meta.yml

```diff
@@ -30,19 +30,37 @@ services:
         io.simcore.contact: '{"contact": "anderegg@itis.swiss"}'
         io.simcore.description: '{"description": "Modern test dynamic service (with
           dynamic sidecar)"}'
-        io.simcore.inputs: '{"inputs": {}}'
+        io.simcore.inputs: '{"inputs": {"string_input": {"displayOrder": 1, "label":
+          "String input", "description": "Test input string with default value", "type":
+          "string", "defaultValue": "lorem ipsum"}, "integer_input": {"displayOrder":
+          2, "label": "Integer input", "description": "Integer field with a default
+          value", "type": "integer", "defaultValue": 100}, "boolean_input": {"displayOrder":
+          3, "label": "Boolean input", "description": "Boolean field with a default
+          value", "type": "boolean", "defaultValue": false}, "number_input": {"displayOrder":
+          4, "label": "Number input", "description": "Number field with a default
+          value", "type": "number", "defaultValue": 1.12}, "file_input": {"displayOrder":
+          5, "label": "File input", "description": "Empty file input", "type": "data:*/*",
+          "fileToKeyMap": {"test_file": "file_input"}}}}'
         io.simcore.integration-version: '{"integration-version": "1.0.0"}'
         io.simcore.key: '{"key": "simcore/services/dynamic/dy-static-file-server-dynamic-sidecar"}'
         io.simcore.name: '{"name": "dy-static-file-server-dynamic-sidecar"}'
-        io.simcore.outputs: '{"outputs": {}}'
+        io.simcore.outputs: '{"outputs": {"string_output": {"displayOrder": 1, "label":
+          "String output", "description": "String value from input", "type": "string"},
+          "integer_output": {"displayOrder": 2, "label": "Integer output", "description":
+          "Integer value from input", "type": "integer"}, "boolean_output": {"displayOrder":
+          3, "label": "Boolean output", "description": "Boolean value from input",
+          "type": "boolean"}, "number_output": {"displayOrder": 4, "label": "Number
+          output", "description": "Number value from input", "type": "number"}, "file_output":
+          {"displayOrder": 5, "label": "File output", "description": "File from input",
+          "type": "data:*/*", "fileToKeyMap": {"test_file": "file_output"}}}}'
         io.simcore.type: '{"type": "dynamic"}'
-        io.simcore.version: '{"version": "1.0.5"}'
+        io.simcore.version: '{"version": "1.0.6"}'
         org.label-schema.build-date: ${BUILD_DATE}
         org.label-schema.schema-version: '1.0'
         org.label-schema.vcs-ref: ${VCS_REF}
         org.label-schema.vcs-url: ${VCS_URL}
-        simcore.service.paths-mapping: '{"outputs_path": "/www/outputs", "inputs_path":
-          "/www/inputs"}'
+        simcore.service.paths-mapping: '{"outputs_path": "/www/inputs-outputs", "inputs_path":
+          "/www/inputs-outputs"}'
```

Comment:
this looks a bit weird... is that correct?

Reply:
Just realised this will have to change. I actually require a more reliable way of moving inputs to the outputs (will do that internally via a script).

At this point will add a python script and a few tests to ensure it is working. I think I will not change the rest of the service.